content/handbook/support/license-and-renewals/workflows/saas/trials_and_plan_change.md

```
@@ -3,110 +3,110 @@ title: GitLab.com のトライアル、延長、プラン変更の取り扱い
+### SFDC で生成された一時的な更新延長
 
-アカウントエグゼクティブ（AE）は、更新案件のクローズに想定以上の時間がかかっている場合に、SalesForce.com（SFDC）を使用して顧客に SaaS の 21 日間サブスクリプション延長を発行できます。AE がこの機能を使用すると、L&R サポートの関与なしにサブスクリプションが自動的に延長されます。[一時的な更新延長](/handbook/product/groups/fulfillment/#temporary-renewal-extensions) のハンドブックエントリにこのアプローチが文書化されています。
+Account Executive (AE) は SalesForce.com (SFDC) を使用して、更新の商談がクローズするまでに予想以上に時間がかかっている場合、お客様に SaaS 21 日間のサブスクリプション延長を発行できます。AE がこの機能を使用すると、L&R サポートの関与なしにサブスクリプションが自動的に延長されます。[Temporary renewal extensions](/handbook/product/groups/fulfillment/#temporary-renewal-extensions) ハンドブックエントリーがこのアプローチを文書化しています。
 
-上記のアプローチには以下の注意点があります:
+なお、上記のアプローチには以下の注意点があります:
 
-1. この機能の悪用を防ぐためのガードレールが設けられています。その結果、更新イベントごとに発行できるサブスクリプション延長は 1 回のみです。そのため、L&R サポートがさらにサブスクリプション延長を生成する必要がある場合があります。これが発生した場合は、[アクティブまたは期限切れのサブスクリプションを延長する](#extend-an-existing-active-or-expired-subscription) の手動プロセスに従ってください。
-1. さらに、サブスクリプションはトライアル以外のサブスクリプションである必要があります。サブスクリプションが今後 15 日以内に期限切れになる予定の場合、`Deviation from SM License Extension Workflow macro` マクロを使用してセールス担当者に SFDC 機能を活用するようリダイレクトし、その後チケットをクローズします。
-1. サブスクリプションの期限切れが 15 日以上先の場合、有効期限が 15 日以内になるまで待ってから SFDC 機能を活用するようセールス担当者に助言します。15 日以内のウィンドウに入ったら、`Deviation from SM License Extension Workflow` マクロを使用してセールス担当者に SFDC 機能を使用するようリダイレクトし、その後チケットをクローズします。
+1. この機能の悪用を防ぐためのガードレールが設けられています。その結果、更新イベントごとに 1 つのサブスクリプション延長しか生成できません。したがって、L&R サポートがさらにサブスクリプション延長を生成する必要がある場合もあります。これが発生した場合は、[アクティブまたは期限切れのサブスクリプションを延長する](#extend-an-existing-active-or-expired-subscription) の手動プロセスに従ってください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)